### ISSUE TITLE
Endtaker

### ISSUE THREAD

**Issue by Korokun** [D3fau4/Helltaker-switch#25]
Are you going to update this to the latest Helltaker DLC?

**Comment by D3fau4** [comment]
which? this?
![](https://cdn.discordapp.com/attachments/519986961382113283/843074349279477770/IMG_20210515_123602.jpg)

**Comment by Korokun** [comment]
Yes! I take it you've already updated Helltaker.nsp to the latest one?

________________________________

**Comment by D3fau4** [comment]
nope

**Comment by Korokun** [comment]
When you do, do it low key you are SUPER appreciated~!!!!

________________________________

**Comment by mikhaelcool7** [comment]
Don't forget the PSVita too ! [People](https://www.reddit.com/r/VitaPiracy/comments/k05w6j/helltaker_switch_vita_port_by_d3fau4/) seem to get a few bugs here and there too

**Comment by D3fau4** [comment]
> Don't forget the PSVita too ! [People](https://www.reddit.com/r/VitaPiracy/comments/k05w6j/helltaker_switch_vita_port_by_d3fau4/) seem to get a few bugs here and there too

https://www.youtube.com/watch?v=U6vqrirKvMA later i explain my problem with vita in reddit

**Comment by D3fau4** [comment]
https://www.reddit.com/r/VitaPiracy/comments/k05w6j/helltaker_switch_vita_port_by_d3fau4/gyvelin/?utm_source=reddit&utm_medium=web2x&context=3

**Comment by Korokun** [comment]
I remember in this post, you needed a **"Unity add-on for Unity 2019.x.xfx"**. Can you tell me what it is (what version if you have yet to get a copy) and how may I send it to you? You can discord me Korokun#0802 if you want to keep it private.

**Comment by MilkTea068** [comment]
Are you adding Examtaker? Or atleast to the latest update? Honestly I think alot of people would appreciate and love It If you would and alot of people (Including me) cannot afford a computer so this Is our alternate source :D thank you for you hard work!
